scheds/rust/scx_bpfland/src/bpf/main.bpf.c

```diff
@@ -166,6 +161,30 @@ static int calloc_cpumask(struct bpf_cpumask **p_cpumask)
 	return 0;
 }
 
+/*
+ * Set the state of a CPU in a cpumask.
+ */
+static bool set_cpu_state(struct bpf_cpumask *cpumask, s32 cpu, bool state)
+{
+	struct bpf_cpumask *mask;
+	int ret = false;
+
+	bpf_rcu_read_lock();
```

Comment:
This isn't necessarily from this PR but what is RCU read lock protecting?

Reply:
Hm.. I had the impression that the verifier was complaining without the RCU protection, but it's not the case (and I also don't see anything to protect here). Probably it was needed to protect something else that was moved after some refactoring. Anyway, thanks for noticing it, I'll drop the RCU read lock here!